src/NLPModelsIpopt.jl

```diff
@@ -181,6 +182,33 @@ function ipopt(nlp::AbstractNLPModel; kwargs...)
   return solve!(solver, nlp, stats; kwargs...)
 end
 
+"""
+    ipopt(nls::AbstractNLSModel; kwargs...)
+
+Solves the `AbstractNLSModel` problem `nls` using `IpOpt` by converting it to a feasibility form.
```

Comment:
"feasibility form" is not really a known name. Maybe something like:
By moving the nonlinear residual to the constraints using slack variables.